lib/cc/analyzer/formatters/formatter.rb

```diff
@@ -2,6 +2,8 @@ module CC
   module Analyzer
     module Formatters
       class Formatter
+        attr_reader :current_engine
```

Comment:
I thought `private` stuff wasn't accessible to subclasses but I could have been wrong.

My main goal here is that this is accessible to subclasses but not outside.